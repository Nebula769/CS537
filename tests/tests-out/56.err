Traceback (most recent call last):
  File "/home/hzheng/private/537/FUSE-File-System/tests/./read-write.py", line 24, in <module>
    fh.write(towrite)
OSError: [Errno 28] No space left on device
