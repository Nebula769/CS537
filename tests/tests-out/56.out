Correct
Correct
total allocated datablocks on all disks: found 36 expected either 12 or 14.
